style/popup.less

```diff
@@ -40,11 +37,25 @@
 	left: 0;
 	top: 0;
 
+	&-top {
+		height: 10vh;
+	}
+
+	&-main {
+		display: flex;
+		max-width: 100%;
+		height: 70vh;
+	}
+
+	&-bottom {
+		height: 20vh;
```

Comment:
I don't find the discussion anywhere, 

1. should the bottom container stay on top of the main? that means main is 100vh.
1. or stay below the main, which the main height should be dynamic based on the size of the caption.

Reply:
discussion : Let's keep it and let design check in here. or revisit in the image attribution ticket.